spec/rubocop/cop/chef/modernize/use_multipackage_installs_spec.rb

```diff
@@ -37,11 +39,31 @@
     expect_correction(<<~RUBY)
       case node['platform']
       when 'ubuntu'
+        include_recipe 'apt'
+
         package %w(bmon htop vim curl)
       end
     RUBY
   end
 
+  it 'does not fail with an empty when statement' do
+    expect_no_offenses(<<~RUBY)
+      case node['platform_family']
+      when 'debian'
+        # nothing here
+      end
+    RUBY
+  end
+
+  it 'does not fail if the when is a regex' do
+    expect_no_offenses(<<~RUBY)
+      case node['platform_family']
+      when /debian/
+        package 'it-is-fine'
```

Comment:
should this test _theoretically_ trigger the rule, in order to be a valid test? LIke:
```
when /debian/
  %w(bmon htop).each { |p| package p }
```
?

Reply:
At the moment we can't handle the regex, but we need to make sure we don't crash.

Reply:
Sure but it seems like we should not-crash in both cases, right? When the package rule is good and bad (regardless of if we do/don't catch the error)

Reply:
Yeah we were crashing before so I added code to not crash on regexes and added a test. Eventually we need another rule to convert unnecessary regexes in case statements, but until that happens we need to just skip these.

Reply:
I get that. I'm just asking, should we add something like:
```
# this is technically an offense, but lets make sure we don't crash
# TODO: Make this "expect_offense" when we fix regex handling.
expect_no_offenses(<<EOF) # is there an expect_no_raise ?
  case node['platform']
  when /debian/
    %w(bmon htop).each { |p| package p }
  end
end
```

just to make sure we're not crashing in that case too?

Reply:
I don't think we'll ever directly fix that here though so it's not really a todo. We'll transform the regexes to strings where we can and then this alert will just magically trigger later.